assets/src/blocks/Counter/CounterFrontend.js

```diff
@@ -21,6 +21,8 @@ export class CounterFrontend extends Component {
     if (completed_api && completed_api.startsWith('https://')) {
       window.addEventListener('updateCounter', function () { counter.calculateRemaining(); }, false);
     }
+    // Append the counter inside the En-Form
+    $('.counter-style-en-forms-bar').appendTo('.enform-extra-header-placeholder').show();
```

Comment:
As we're trying to remove jQuery from the frontend, can you do it in vanilla JS ? something like (didn't test :grimacing:)
```js
let counter_placeholder = document.querySelector('.enform-extra-header-placeholder');
let counter = document.querySelector('.counter-style-en-forms-bar');
if (counter_placeholder && counter) {
    counter_placeholder.appendChild(counter);
}
```
If you do you can close this one too https://jira.greenpeace.org/browse/PLANET-5663 :partying_face:

Reply:
This one I think :grin: https://jira.greenpeace.org/browse/PLANET-5766